flag/models.py

```diff
@@ -121,7 +121,7 @@ class FlaggedContent(models.Model):
                                   null=True,
                                   related_name="moderated_content")
     count = models.PositiveIntegerField(default=0)
-    when_updated = models.DateTimeField(auto_now=True, auto_now_add=True)
```

Comment:
@gEndelf because i got the error 
flag.FlaggedContent.when_updated: (fields.E160) The options auto_now, auto_now_add, and default are mutually exclusive. Only one of these options may be present.